be/src/storage/rowset/rowset_writer.cpp

```diff
@@ -76,10 +76,10 @@ ssize_t SegmentFileWriter::WriteV(const iovec* iov, int iovcnt) {
     }
 }
 
-BetaRowsetWriter::BetaRowsetWriter(const RowsetWriterContext& context)
+RowsetWriter::RowsetWriter(const RowsetWriterContext& context)
         : _context(context), _num_rows_written(0), _total_row_size(0), _total_data_size(0), _total_index_size(0) {}
 
-Status BetaRowsetWriter::init() {
+Status RowsetWriter::init() {
```

Comment:
In line 98, `_rowset_meta_pb->set_rowset_type(BETA_ROWSET);`
you can also remove it.
As well remove the BETA_ROWSET concept in gensrc/ directory and FE.

Reply:
https://github.com/StarRocks/starrocks/pull/7861